xbmc/windowing/gbm/drm/DRMUtils.h

```diff
@@ -47,7 +47,7 @@ class CDRMUtils
 
   int GetFileDescriptor() const { return m_fd; }
   int GetRenderNodeFileDescriptor() const { return m_renderFd; }
-  const char* GetRenderDevicePath() const { return m_renderDevicePath; }
+  const char* GetRenderDevicePath() const { return m_renderDevicePath.get(); }
```

Comment:
```suggestion
  const char* GetRenderDevicePath() const { return m_renderDevicePath.c_str(); }
```

Reply:
The libdrm functions return a strdup() allocated string, this is equivalent to c_str() from the std::string class.